packages/block-library/src/navigation-menu/index.php

```diff
@@ -74,15 +79,15 @@ function render_block_navigation_menu( $attributes, $content, $block ) {
 	if ( array_key_exists( 'textColorValue', $attributes ) ) {
 		$comp_inline_styles .= ' color: ' . esc_attr( $attributes['textColorValue'] ) . ';';
 	}
-	$comp_inline_styles = ! empty( $comp_inline_styles )
-		? ' style="' . esc_attr( trim( $comp_inline_styles ) ) . '"'
-		: '';
 
 	$colors = build_css_colors( $attributes );
 
-	return "<nav class='wp-block-navigation-menu' {$comp_inline_styles}>" .
-		build_navigation_menu_html( $block, $colors ) .
-	'</nav>';
+	return sprintf(
+		'<nav class="%1$s" style="%2$s">%3$s</nav>',
+		esc_attr( $classes ),
+		esc_attr( trim( $comp_inline_styles ) ),
```

Comment:
We could do the `trim()` operation outside this `sprintf` and then pass the whole attribute as `%2`. Avoids the empty `style=""`.